private/private.h

```diff
@@ -32,7 +32,9 @@
 #include <os/availability.h>
 #include <TargetConditionals.h>
 #include <os/base.h>
-#elif defined(__linux__) || defined(__FreeBSD__)
+#elif defined(_WIN32)
```

Comment:
this should use `TARGET_OS_WIN32` to be consistent with the rest of the tests I think